code/modules/mob/living/silicon/silicon.dm

```diff
@@ -6,6 +6,8 @@
 	var/lawchannel = MAIN_CHANNEL // Default channel on which to state laws
 	var/list/stating_laws = list()// Channels laws are currently being stated on
 	var/obj/item/device/radio/common_radio
+	//plug before baymed arrives
+	var/obj/item/device/radio/silicon_radio
```

Comment:
BTW, that seem so redundant

Reply:
this will stay, baymed renamed it but it used in nano module for checks. If i fix it now i will have to later undo it